content/en/platform/corda/5.0-dev-preview-2/introduction/corda4-differences.md

```diff
@@ -11,53 +11,52 @@ section_menu: corda-5-dev-preview2
 
-Corda 5 uses the Gateway (and session manager) to achieve similar goals. However, now that Corda communicates with other clusters via HTTPS instead of AMQP, these services can be deployed into an ingress zone behind the corporate firewall and load balancing proxies.
+Corda 5 uses the Gateway (and session manager) to achieve similar goals. However, now that Corda communicates with other clusters via HTTPS instead of AMQP, these services can be deployed into an ingress zone behind the corporate firewall and load-balancing proxies.
 
 ## All-in-One Worker
-The worker architure allows us to deliver Corda in various guises. For highly available, fault tolerant systems, the fully distributed collection of workers can be used. However, when developing CorDapps, the ability to run a small, lightweight version of this is paramount. Therefore, a collection of workers can be collapsed into a single all-in-one worker, delivering the same characteristics of the mode complex deployment in a much smaller package.
+The worker architecture allows us to deliver Corda in various guises. For highly available fault-tolerant systems, the fully distributed collection of workers can be used. However, when developing CorDapps, the ability to run a small and lightweight version of this is paramount. Therefore, a collection of workers can be collapsed into a single all-in-one worker, delivering the same characteristics of the mode-complex deployment in a much smaller package.
 
 This can be augmented through the addition of an in-memory message bus, foresaking Kafka. This allows deterministic testing and better observability when running through the full software development lifecycle on a laptop.
 
 ## Pluggable Ledger Model
-Past Corda versions tightly embedded the UTXO ledger model at the heart of the architecture. Corda 5, with its layers approach, makes this model pluggable. This means various ledger models, from UTXO through Xonasentual to some form of rules-based implementation, are all possible to deploy on top of the same tech stack and run in parallel. The right tool for the job can be selected rather than requiring the full set of UTXO and smart contract features, which can overcomplicate applications that do not mesh well with that paradigm.
\ No newline at end of file
+Past Corda versions tightly embedded the UTXO ledger model at the heart of the architecture. Corda 5, with its layered approach, makes this model pluggable. This means that various ledger models, from UTXO through Xonasentual, to some form of rules-based implementation, are all possible to deploy on top of the same tech stack and run in parallel. The right tool for the job can be selected, rather than requiring the full set of UTXO and smart contract features, which can overcomplicate applications that do not mesh well with that paradigm.
```

Comment:
```suggestion
Past Corda versions tightly embedded the UTXO ledger model at the heart of the architecture. Corda 5, with its layered approach, makes this model pluggable. This means that various ledger models, from UTXO through consensual, to some form of rules-based implementation, are all possible to deploy on top of the same tech stack and run in parallel. The right tool for the job can be selected, rather than requiring the full set of UTXO and smart contract features, which can overcomplicate applications that do not mesh well with that paradigm.
```